OptionSelector/CQDetailTableViewController.h

```diff
@@ -0,0 +1,15 @@
+//
+//  CQDetailTableViewController.h
+//  Unit1WeekFourHomework
+//
+//  Created by Jackie Meggesto on 8/12/15.
+//  Copyright (c) 2015 Jackie Meggesto. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "CQCategory.h"
```

Comment:
Style: You should try to keep your `#import` directives in .m files. You can get around having it here by replacing it with the following:
```objc
@class QCCategory;
```